tests/index.ts

```diff
@@ -31,4 +31,12 @@ describe('fuuid', () => {
       .toBeGreaterThan(getVariationCount(UUID_CHAR_COUNT, UUID_LENGTH));
   });
 
+  it('should contain only valid terms', () => {
+    const id = fuuid();
+    const words = id.split('-');
+    expect(words).toHaveLength(LENGTH);
```

Comment:
The regex test already checks this.

Reply:
Yes, but if it's an empty list, then the below `forEach` will still pass. Could've just done truthy check, but this covers it better IMO

Reply:
Meh, guess you can't be too careful. :P